### ISSUE TITLE
RFE: API for Detecting Failed Action in Workflow

### ISSUE THREAD

**Issue by rmathew** [YahooArchive/oozie#535]
When a work-flow fails, the user has to manually scan the logs to determine the failing action and (after having taken appropriate measures) re-submit the work-flow starting at the failed action.

Provide an automated mechanism for detecting the failed action in an instance. This could be used by wrapper scripts or Jobs to, for example, work around transient failures without involving manual intervention.

**Comment by anew** [comment]
Issue 78 (https://github.com/yahoo/oozie/issues/78) added the ability to rerun a workflow from the failed node (without having to specify which nodes to skip). Does that address your need? If so, I would like to close this as a dup.

**Comment by rmathew** [comment]
Yes, it does look like what I'm asking for here. From what I can make out, I'd be able to submit a work-flow instance via the Java API and then re-submit it upon failure using the API and setting an additional configuration property.
